ChangeLog.md

```diff
@@ -1,3 +1,29 @@
+# ooniprobe-ios 1.1.3 [2017-03-08]
+
+Bugfixes:
+
+* Various GUI fixing
+
+* Expose app version number in app
+
+* Empty screen in "Past Tests" on init
+
+* Delete all tests button
+
+* Render failed "NDT Speed Test" 
+
+New languages:
+
+* IT
+
+* FR
+
+* ES
```

Comment:
I would maybe put this at the top of the Changelog as a new feature.

New Features:

* Added multilingual support and Spanish, French and Italian translations